src/hexo/themes/sonarwhal/source/core/css/navigation.css

```diff
@@ -127,23 +182,47 @@ button.mobile-nav-button {
     .nav__navbar.show {
         position: absolute;
         display: block;
-        top: 8rem;
+        top: 9.4rem;
         left: 0;
-        background-color: #4046dd;
+        margin: 0;
+        background-color: #3626a3;
         z-index: 30;
     }
 
     .nav-bar .navbar__submenu > li > a {
         padding-left: 2.3rem;
     }
 
+    .nav-bar--container {
+        display: none;
+    }
+
+    .nav-bar--container.show {
```

Comment:
@qzhou1607 when this button: `<button class="header__toggle mobile-search-button" aria-label="Search"></button>` is clicked by the user, the class "show" needs to be added to the div: <div class="nav-bar--container">

Should be the same functionality as the menu in mobile view. Let me know if this makes sense.